### PR TITLE
zeppelin setup

### PR DESCRIPTION
**Pull request by fparisotto**

### CODE REVIEW

File: remote_hook.sh [hunk 1]
```diff
@@ -49,6 +49,23 @@ on_trap_exit() {
     rm -f "${RUNNING_FILE}"
 }
 
+install_and_run_zeppelin() {
+    if [[ ! -d "zeppelin" ]]; then
+        wget "http://www.us.apache.org/dist/incubator/zeppelin/0.5.6-incubating/zeppelin-0.5.6-incubating-bin-all.tgz" -O zeppelin.tar.gz
+        tar xvzf zeppelin.tar.gz > /tmp/zeppelin_install.log
```
Comment (douglaz) [inline remote_hook.sh]: Alternatively, you could do:
mkdir zepplin
tar xvzf zeppelin.tar.gz -C zeppelin --strip-components 1 > /tmp/zeppelin_install.log

File: remote_hook.sh [hunk 2]
```diff
@@ -49,6 +49,23 @@ on_trap_exit() {
...
+        mv `ls -d zeppelin-*` zeppelin
+    fi
+    if [[ -f "zeppelin/bin/zeppelin.sh" ]]; then
+        export MASTER="${JOB_MASTER}"
+        export ZEPPELIN_PORT="8081"
+        export SPARK_HOME="/root/spark"
+        export SPARK_SUBMIT_OPTIONS="--jars ${JAR_PATH}"
+        sudo -E zeppelin/bin/zeppelin.sh
+    else
+        notify_error_and_exit "Not found zeppelin installation"
```
Comment (douglaz) [inline remote_hook.sh]: Zepellin installation not found

Comment (douglaz) [inline remote_hook.sh]: One simple but killer feature would be to launch the browser in the correct host and port (through cluster.py).

**Comment by douglaz** [comment]
One simple but killer feature would be to launch the browser in the correct host and port (through cluster.py).

**Comment by sisso** [comment]
LGTM

I didn't decide between scary or amazing

File: tools/cluster.py [hunk 3]
```diff
@@ -436,6 +436,9 @@ def job_run(cluster_name, job_name, job_mem,
                src_local=remote_hook_local,
                remote_path=with_leading_slash(remote_path))
 
+    if job_name == "zeppelin":
+         subprocess.Popen(["xdg-open", "http://{master}:8081".format(master=master)])
```
Comment (douglaz) [inline tools/cluster.py]: Have you tried 
https://docs.python.org/2/library/webbrowser.html?

**Comment by douglaz** [comment]
LGTM.

**Pull request merged**